streaming/src/main/scala/io/finch/streaming/package.scala

```diff
@@ -0,0 +1,49 @@
+package io.finch
+
+import com.twitter.io.{Buf, Reader}
+import com.twitter.util.Future
+import io.catbird.util._
+import io.finch.Endpoint.Result
+import io.finch.items.RequestItem
+import io.iteratee.{Enumeratee, Enumerator}
+
+/**
+  * Streaming module
+  */
+package object streaming {

```

Comment:
Well, I guess it would save few symbols when lifting into a `Enumerator` but also add some overhead here and there, such as defining a monad, instead of picking it from imports.
Not sure if it worth it.